qiita_db/study.py

```diff
@@ -789,6 +796,87 @@ def _portals(self):
             TRN.add(sql, [self._id])
             return TRN.execute_fetchflatten()
 
+    @property
+    def ebi_study_accession(self):
+        """The EBI study accession for this study
+
+        Returns
+        -------
+        str
+            The study EBI accession
+        """
+        with TRN:
+            sql = """SELECT ebi_study_accession
+                     FROM qiita.{0}
+                     WHERE study_id = %s""".format(self._table)
+            TRN.add(sql, [self._id])
+            return TRN.execute_fetchlast()
+
+    @ebi_study_accession.setter
+    def ebi_study_accession(self, value):
+        """Sets the study's EBI study accession
+
+        Parameters
+        ----------
+        value : str
+            The new EBI study accession
+
+        Raises
+        ------
+        QiitDBError
+            If the study already has an EBI study accession
+        """
+        with TRN:
+            if self.ebi_study_accession is not None:
```

Comment:
Since `self.ebi_study_accession` is accession to the database, by including it in the TRN block will make sure that such operation is executed inside the transaction of the setter, avoiding potential lost-write errors in a multi-user environment.